ui/src/core/index.ts

```diff
@@ -126,19 +126,38 @@ export function generateCore() {
 			*/
 
 			const accessor = parseAccessor(key);
+			let lastElementIndex = accessor.length - 1
 			let stateRef = userState.value;
-			for (let i = 0; i < accessor.length - 1; i++) {
+			
+			// Check if the accessor is meant for deletion.
+			const isDeletion = accessor[lastElementIndex].charAt(0) === '-';
```

Comment:
Can we instead apply what we discussed earlier i.e. reserving the first character of the accessor to indicate the type of operation? i.e. + addition (perhaps change), - (deletion).

I don't want someone to break it when adding a minus to their key name nor I'd like to implement character escaping logic beyond the dots.

Reply:
My bad, apologies and thank you for noticing. I actually made it that way, but then got for whatever reason distracted and "removed" it for PR, though I shouldn't have had. I'll fix this.